src/streams/openssl.c

```diff
@@ -627,6 +627,16 @@ int git_openssl_stream_new(git_stream **out, const char *host, const char *port)
 	return error;
 }
 
+int git_openssl_set_cert_file(const char *file, const char *path)
+{
+	if (SSL_CTX_load_verify_locations(git__ssl_ctx, file, path) == 0) {
+		giterr_set(GITERR_SSL, "SSL error: %s",
```

Comment:
Maybe be a bit more specific here? E.g. "could not set default locations for certificates"?